bpf/init.sh

```diff
@@ -16,18 +16,13 @@
 
 LIB=$1
 RUNDIR=$2
-IP6_ROUTER=$3
-IP4_HOST=$4
-IP6_HOST=$5
-IP4_RANGE=$6
-IP6_RANGE=$7
-IP4_SVC_RANGE=$8
-IP6_SVC_RANGE=$9
-MODE=${10}
+IP4_HOST=$3
```

Comment:
Not too relevant, but we could use `shift` to avoid renumbering args on refactor. This doesn't happen too often, though.

Reply:
@scanf is working on removing `init.sh` entirely in favor of a golang native implementation.